<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Pig-gene by genepi</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1>Pig-gene</h1>
        <p>Due to the vast progress in sequencing, data volumes in bioinformatics are increasing dramatically. Therefore, the aim of this work is the development of a graphical generation platform for Apache Pig scripts called PigGene to analyze data in a massively parallel way. Users are guided through the building process of scripts by an intuitive user interface and are able to generate workflow definitions for complex data queries on the fly. They can also re-use existing workflows within a new workflow definition. Generated scripts are ready to use and can be further executed directly on Cloudgene, a graphical MapReduce execution platform. PigGene supports especially inexperienced users in the area of bioinformatics.</p>

        <p class="view"><a href="https://github.com/genepi/pig-gene">View the Project on GitHub <small>genepi/pig-gene</small></a></p>


        <ul>
          <li><a href="https://github.com/genepi/pig-gene/zipball/master">Download <strong>ZIP File</strong></a></li>
          <li><a href="https://github.com/genepi/pig-gene/tarball/master">Download <strong>TAR Ball</strong></a></li>
          <li><a href="https://github.com/genepi/pig-gene">View On <strong>GitHub</strong></a></li>
        </ul>
      </header>
      <section>
        <p><img src="https://github.com/genepi/pig-gene/screenshot.png" alt="pig-gene screenshot"></p>

<p>This screenshot depicts the interface of the pig-gene application. Users can select workflow-related actions in the navigation bar on top of the window. Below is the workflow definition which shows the name and the description of the workflow as well as the defined input and output variables. These are useful for the user so that he can reuse loaded files if he wants to. A tabular representation of the workflow is depicted between the input and output variable definitions. The user has the possibility to add single step operations like a filter-operation or to add an already defined workflow as a step within the new workflow definition (work in progress). It is also possible to edit or remove steps from the definition. Reordering of workflow steps will be added within the next weeks as well.</p>

<p><strong>authors and contributors</strong>
Clemens Banas (<a href="https://github.com/clembanas" class="user-mention">@clembanas</a>), Lukas Forer (<a href="https://github.com/lukfor" class="user-mention">@lukfor</a>) and Sebastian Sch√∂nherr (<a href="https://github.com/seppinho" class="user-mention">@seppinho</a>)</p>

<p><strong>info</strong>
Please note that this project is work in progress and some functionality is not implemented yet. It will get updated within the following weeks. (last edited: Friday, April 4, 2014)</p>
      </section>
      <footer>
        <p>This project is maintained by <a href="https://github.com/genepi">genepi</a></p>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script src="javascripts/scale.fix.js"></script>
    
  </body>
</html>